ext/featurefmt/featurefmttest/featurefmttest.go

```diff
@@ -0,0 +1,160 @@
+// Package featurefmttest contains utility functions for featurefmt tests
+package featurefmttest
+
+import (
+	"bytes"
+	"encoding/csv"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coreos/clair/ext/featurefmt"
+
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/pkg/tarutil"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestData represents the data used to test an implementation of Lister.
```

Comment:
either we'll keep this for small instances of package information testing, or we migrate to the new way of testing, which I prefer for larger sample.